samples/sensor/paj7620_gesture_polling/sample.yaml

```diff
@@ -0,0 +1,9 @@
+sample:
```

Comment:
Make this one sample for the sensor with a Kconfig maybe and just add two tests?